lua/conform/runner.lua

```diff
@@ -49,10 +52,13 @@ M.build_cmd = function(formatter_name, ctx, config)
       :gsub("$DIRNAME", ctx.dirname)
       :gsub("$RELATIVE_FILEPATH", compute_relative_filepath)
       :gsub("$EXTENSION", ctx.filename:match(".*(%..*)$") or "")
-    return command .. " " .. interpolated
+    return {
+      vim.split(vim.o.shell, " "),
+      vim.split(vim.o.shellcmdflag, " "),
+      string.format("{%s %s}", command, interpolated),
+    }
```

Comment:
I think this works in many cases, but isn't quite the same as what `jobstart` was doing before. I'd recommend checking out what's going on under the hood and trying to replicate that in lua. Some pointers:
- [jobstart builds the argument list](https://github.com/neovim/neovim/blob/27fb62988e922c2739035f477f93cc052a4fee1e/src/nvim/eval/funcs.c#L4248)
- [handle the case where the args are a string](https://github.com/neovim/neovim/blob/27fb62988e922c2739035f477f93cc052a4fee1e/src/nvim/eval.c#L5847-L5853)
- [function that builds the arg list](https://github.com/neovim/neovim/blob/27fb62988e922c2739035f477f93cc052a4fee1e/src/nvim/os/shell.c#L583-L606)

I expect this will be complicated enough to need its own utility function, with associated unit tests.

Reply:
I realized I can just pull the function directly from C, so I don't think we need unit tests. Thanks for the guidance!